sched/init/nx_start.c

```diff
@@ -522,6 +521,8 @@ void nx_start(void)
         }
     }
 
+  g_lastpid = cpu ? cpu -1 : 0;
```

Comment:
```suggestion
  g_lastpid = cpu ? cpu - 1 : 0;
```
nit: missing whitespace

Reply:
> 
> 
> nit: missing whitespace

Done. thank you.